### PR TITLE
Add tag for replacing certificate

### PR DESCRIPTION
**Pull request by b10n**
This adds a tag to selectively run the tasks for regenerating certificates.

### CODE REVIEW

File: tasks/easyrsa.yml [hunk 1]
```diff
@@ -24,7 +24,11 @@
 - include_tasks: dh.yml
   when: easyrsa_generate_dh | bool
 
-- include_tasks: download.yml
+- include_tasks:
+    file: download.yml
+    apply:
+      tags: easyrsa-replace
```
Comment (nkakouros) [inline tasks/easyrsa.yml:30]: Why do we need this syntax here (`apply`) but not in the previous `include_tasks`?

Comment (b10n) [reply tasks/easyrsa.yml:30]: I wish I had an answer to that. Individual `include_tasks` won't fire without it, but `block`s do.

Comment (nkakouros) [reply tasks/easyrsa.yml:30]: Wow, interesting.

**Comment by nkakouros** [comment]
Thanks @b10n !

**Pull request merged**